ulift-web/src/Pages/NotFound/index.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import Lottie from "react-lottie";
+import { Link } from "react-router-dom";
+
+import "./styles.css";
+import Logo from "../../assets/logo.svg";
+import Animation from "../../assets/404.json";
+
+const Header = () => {
```

Comment:
Esse Header vai ser usado em outros componentes? se for, por que não criar um componente só pra ele para poder ser reaproveitado depois?

Reply:
Este vai ser utilizado somente nesta página, por isso criei dentro da mesma. O header que vai se utilizar nas outras ainda vai ser criado e colocado dentro da pasta de componentes